src/app/public/modules/repeater/repeater.service.ts

```diff
@@ -29,6 +29,8 @@ export class SkyRepeaterService implements OnDestroy {
 
   public items: SkyRepeaterItemComponent[] = [];
 
+  public orderChange = new EventEmitter<void>();
```

Comment:
Since this is in a service and not a component, should this be a `Subject` or `BehaviorSubject` rather than an `EventEmitter`?

Reply:
Good call - I agree. Updated to a `BehaviorSubject`.